src/detect-detection-filter.c

```diff
@@ -302,14 +302,10 @@ static int DetectDetectionFilterTestParse01 (void)
  */
 static int DetectDetectionFilterTestParse02 (void)
 {
-    DetectThresholdData *df = NULL;
-    df = DetectDetectionFilterParse("track both,count 10,seconds 60");
-    if (df && (df->track == TRACK_DST || df->track == TRACK_SRC) && (df->count == 10) && (df->seconds == 60)) {
-        DetectDetectionFilterFree(NULL, df);
-        return 0;
-    }
+    DetectThresholdData *df = DetectDetectionFilterParse("track both,count 10,seconds 60");
+    FAIL_IF_NOT_NULL(df);
```

Comment:
Aren't we missing some FAIL checks, here?

Reply:
I stand corrected :P https://github.com/OISF/suricata/pull/8088#discussion_r1004861760